coreos-base/oem-packet/files/units/packet-phone-home.service

```diff
@@ -6,8 +6,9 @@ After=coreos-metadata.service
 
 [Service]
 EnvironmentFile=/run/metadata/coreos
-Type=oneshot
```

Comment:
Does anything order itself `After` this service? The change from `oneshot` to `simple` will change when the dependent service runs.